config/cruisecontrol.properties

```diff
@@ -101,7 +101,7 @@ goals=com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal,com.linkedin
 # The list of supported intra-broker goals
 intra.broker.goals=com.linkedin.kafka.cruisecontrol.analyzer.goals.IntraBrokerDiskCapacityGoal,com.linkedin.kafka.cruisecontrol.analyzer.goals.IntraBrokerDiskUsageDistributionGoal
 
-# The list of supported hard goals -- consider using RackAwareDistributionGoal instead of RackAwareGoal in clusters with partitions whose replication factor > number of racks
+# The list of supported hard goals -- consider using RackAwareDistributionGoal instead of RackAwareGoal in clusters with partitions whose replication factor > number of racks. The value must be a subset of the "goals" and a superset of the "hard.goals" and "self.healing.goals".
```

Comment:
Nit: `...and a superset of the "hard.goals" and "self.healing.goals"` -> `...and a superset of the "self.healing.goals"`?

Reply:
Hi Efe, do you mean it's not a superset of the hard.goals?

Reply:
This documentation is for `hard.goals` -- and it implies that the value of `hard.goals` must be a superset of the "hard.goals" (?).

Am I missing something?

Reply:
My bad. Those comments should be for default goals. Will fix